clients/python/README.md

```diff
@@ -185,16 +185,33 @@ Currently, the Model Registry backend treats model lists as a circular buffer, a
 
 ## Development
 
+### Using the Makefile
+
+The `Makefile` contains most common development tasks
+
+To install dependencies:
+
+```bash
+make
+```
+
+Then you can run tests:
+
+```bash
+make test test-e2e
+```
+
+### Using Nox
+
 Common tasks, such as building documentation and running tests, can be executed using [`nox`](https://github.com/wntrblm/nox) sessions.
 
 Use `nox -l` to list sessions and execute them using `nox -s [session]`.
 
-Alternatively, use `make install` to setup a local Python virtual environment with `poetry`.
+### Testing requirements
 
 To run the tests you will need `docker` (or equivalent) and the `compose` extension command.
```

Comment:
I think we can change this section too, since we are now using kind instead of docker compose